Bug: pnpm format 出现了许多warning，这些代码和样例的一模一样，为什么样例没有warning？

pnpm format

> vben-admin-monorepo@5.5.9 format /Users/zgs/Documents/ws/HiAuth/hiauth-front
> vsh lint --format

(node:2143) ExperimentalWarning: Importing JSON modules is an experimental feature and might change at any time
(Use `node --trace-warnings ...` to show where the warning was created)

/Users/zgs/Documents/ws/HiAuth/hiauth-front/apps/web-antd/src/adapter/component/index.ts
   92:26  warning  There is more than one component in this file  vue/one-component-per-file
  262:26  warning  There is more than one component in this file  vue/one-component-per-file

/Users/zgs/Documents/ws/HiAuth/hiauth-front/apps/web-auth/src/adapter/component/index.ts
   92:26  warning  There is more than one component in this file  vue/one-component-per-file
  262:26  warning  There is more than one component in this file  vue/one-component-per-file

/Users/zgs/Documents/ws/HiAuth/hiauth-front/packages/@core/preferences/src/preferences.ts
  66:1  warning  Missing @param "namespace.namespace"  jsdoc/check-param-names
  66:1  warning  Missing @param "namespace.overrides"  jsdoc/check-param-names

/Users/zgs/Documents/ws/HiAuth/hiauth-front/playground/src/adapter/component/index.ts
   92:26  warning  There is more than one component in this file  vue/one-component-per-file
  271:26  warning  There is more than one component in this file  vue/one-component-per-file

✖ 8 problems (0 errors, 8 warnings)



---------------------

  System:
    OS: macOS 15.4.1
    CPU: (8) arm64 Apple M1
    Memory: 177.22 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.15.0 - /Users/zgs/.nvm/versions/node/v20.15.0/bin/node
    npm: 10.7.0 - /Users/zgs/.nvm/versions/node/v20.15.0/bin/npm
    pnpm: 10.22.0 - /Users/zgs/.nvm/versions/node/v20.15.0/bin/pnpm
  Browsers:
    Chrome: 143.0.7499.170
    Safari: 18.4

--- a/apps/web-antd/src/adapter/component/index.ts
+++ b/apps/web-antd/src/adapter/component/index.ts
@@ -3,6 +3,8 @@
  * 可用于 vben-form、vben-modal、vben-drawer 等组件使用,
  */
 
+/* eslint-disable vue/one-component-per-file */
+
 import type {
   UploadChangeParam,
   UploadFile,

--- a/packages/@core/composables/src/__tests__/use-sortable.test.ts
+++ b/packages/@core/composables/src/__tests__/use-sortable.test.ts
@@ -29,9 +29,8 @@ describe('useSortable', () => {
     await initializeSortable();
 
     // Import sortablejs to access the mocked create function
-    const Sortable = await import(
-      'sortablejs/modular/sortable.complete.esm.js'
-    );
+    const Sortable =
+      await import('sortablejs/modular/sortable.complete.esm.js');
 
     // Verify that Sortable.create was called with the correct parameters
     expect(Sortable.default.create).toHaveBeenCalledTimes(1);

--- a/packages/@core/preferences/src/preferences.ts
+++ b/packages/@core/preferences/src/preferences.ts
@@ -63,8 +63,9 @@ class PreferenceManager {
 
   /**
    * 初始化偏好设置
-   * @param namespace - 命名空间，用于隔离不同应用的配置
-   * @param overrides - 要覆盖的偏好设置
+   * @param options - 初始化配置项
+   * @param options.namespace - 命名空间，用于隔离不同应用的配置
+   * @param options.overrides - 要覆盖的偏好设置
    */
   initPreferences = async ({ namespace, overrides }: InitialOptions) => {
     // 防止重复初始化

--- a/packages/@core/ui-kit/form-ui/src/types.ts
+++ b/packages/@core/ui-kit/form-ui/src/types.ts
@@ -350,9 +350,9 @@ export interface ActionButtonOptions extends VbenButtonProps {
 export interface VbenFormProps<
   T extends BaseFormComponentType = BaseFormComponentType,
 > extends Omit<
-    FormRenderProps<T>,
-    'componentBindEventMap' | 'componentMap' | 'form'
-  > {
+  FormRenderProps<T>,
+  'componentBindEventMap' | 'componentMap' | 'form'
+> {
   /**
    * 操作按钮是否反转（提交按钮前置）
    */

--- a/packages/@core/ui-kit/shadcn-ui/src/components/button/button.ts
+++ b/packages/@core/ui-kit/shadcn-ui/src/components/button/button.ts
@@ -27,8 +27,10 @@ export type CustomRenderType = (() => Component | string) | string;
 
 export type ValueType = boolean | number | string;
 
-export interface VbenButtonGroupProps
-  extends Pick<VbenButtonProps, 'disabled'> {
+export interface VbenButtonGroupProps extends Pick<
+  VbenButtonProps,
+  'disabled'
+> {
   /** 单选模式下允许清除选中 */
   allowClear?: boolean;
   /** 值改变前的回调 */

--- a/packages/effects/common-ui/src/components/captcha/types.ts
+++ b/packages/effects/common-ui/src/components/captcha/types.ts
@@ -54,8 +54,7 @@ export interface PointSelectionCaptchaCardProps {
   width?: number | string;
 }
 
-export interface PointSelectionCaptchaProps
-  extends PointSelectionCaptchaCardProps {
+export interface PointSelectionCaptchaProps extends PointSelectionCaptchaCardProps {
   /**
    * 是否展示确定按钮
    * @default false

--- a/playground/src/adapter/component/index.ts
+++ b/playground/src/adapter/component/index.ts
@@ -3,6 +3,8 @@
  * 可用于 vben-form、vben-modal、vben-drawer 等组件使用,
  */
 
+/* eslint-disable vue/one-component-per-file */
+
 import type {
   UploadChangeParam,
   UploadFile,

--- a/playground/src/router/routes/modules/demos.ts
+++ b/playground/src/router/routes/modules/demos.ts
@@ -157,9 +157,7 @@ const routes: RouteRecordRaw[] = [
                 name: 'HideChildrenInMenuDemo',
                 path: '',
                 component: () =>
-                  import(
-                    '#/views/demos/features/hide-menu-children/parent.vue'
-                  ),
+                  import('#/views/demos/features/hide-menu-children/parent.vue'),
                 meta: {
                   // hideInMenu: true,
                   title: $t('demos.features.hideChildrenInMenu'),
@@ -169,9 +167,7 @@ const routes: RouteRecordRaw[] = [
                 name: 'HideChildrenInMenuChildrenDemo',
                 path: '/demos/features/hide-menu-children/children',
                 component: () =>
-                  import(
-                    '#/views/demos/features/hide-menu-children/children.vue'
-                  ),
+                  import('#/views/demos/features/hide-menu-children/children.vue'),
                 meta: {
                   activePath: '/demos/features/hide-menu-children',
                   title: $t('demos.features.hideChildrenInMenu'),
@@ -247,9 +243,7 @@ const routes: RouteRecordRaw[] = [
             name: 'RequestParamsSerializerDemo',
             path: '/demos/features/request-params-serializer',
             component: () =>
-              import(
-                '#/views/demos/features/request-params-serializer/index.vue'
-              ),
+              import('#/views/demos/features/request-params-serializer/index.vue'),
             meta: {
               icon: 'lucide:git-pull-request-arrow',
               title: $t('demos.features.requestParamsSerializer'),

--- a/playground/src/views/examples/context-menu/index.vue
+++ b/playground/src/views/examples/context-menu/index.vue
@@ -43,15 +43,14 @@ const contextMenus = () => {
     },
   ];
 };
-
 </script>
 
 <template>
   <Page title="Context Menu 上下文菜单">
     <Card title="基本使用">
       <div>一共四个菜单（刷新、关闭当前、关闭其他、关闭所有）</div>
-      <br/>
-      <br/>
+      <br />
+      <br />
       <VbenContextMenu :menus="contextMenus" :modal="true" item-class="pr-6">
         <Button> 右键点击我打开上下文菜单(有隐藏项) </Button>
       </VbenContextMenu>

